cli/command/system/info.go

```diff
@@ -261,7 +262,19 @@ func printSwarmInfo(dockerCli command.Cli, info types.Info) {
 		fmt.Fprintln(dockerCli.Out(), " ClusterID:", info.Swarm.Cluster.ID)
 		fmt.Fprintln(dockerCli.Out(), " Managers:", info.Swarm.Managers)
 		fmt.Fprintln(dockerCli.Out(), " Nodes:", info.Swarm.Nodes)
+		var strAddrPool strings.Builder
+		if info.Swarm.Cluster.DefaultAddrPool != nil {
+			for _, p := range info.Swarm.Cluster.DefaultAddrPool {
+				strAddrPool.WriteString(p + "  ")
```

Comment:
Perhaps we should use the same formatting as is used for manager IP-addresses here (one pool per line); https://github.com/docker/cli/blob/fd060d2f337a2698797836ebfdae74ae2e604869/cli/command/system/info.go#L300-L304

Reply:
there were other existing examples had similar format. Hence I followed the same. if you want I can change that too.
Docker info
 Network: bridge host macvlan null overlay
 Log: awslogs fluentd gcplogs gelf journald json-file logentries splunk syslog

Reply:
It's not a blocker, we can discuss later, and do a follow up if we think the other format is better